### PR TITLE
Module

### PR DESCRIPTION
**Pull request by DavyKn**
- [ ] npm packages introduced.
- [ ] JS files loading in MODULE
- [ ] Date displayed using LUXON

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import Book from './module/books.js';
+import { bookForm, books } from './module/intro.js';
+import { DateTime } from './module/luxon.js';
+
+books.forEach(Book.createBook);
+bookForm.onsubmit = (e) => {
+  e.preventDefault();
+  const newBook = Book.addBook(new Book(bookForm.title.value, bookForm.author.value));
+  Book.createBook(newBook);
+};
```
Comment (codepantha) [inline index.js]: - [x] When a new book is added, there is no interaction to show the user that anything has happened. Please empty the inputs when the "Add" button is clicked.
The image below shows that the Book details are still there even after Adding the book 👇 
image

- [ ] [OPTIONAL] Optionally, add a success response so the user knows that they have successfully added a book

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUESTED CHANGES ♻️ 

## Hi @DavyKn 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### HIghlights 🎯 
✔️ Good commit history
✔️ Linter checks
✔️ App working almost as expected

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (DavyKn) [reply index.js]: - [ ] The inputs are emptied after the book is added.

**Review by uwadonat — APPROVED** [review]
## Hi @DavyKn,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
appr

# Status: Approved ✔️✔️✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- To add a title on your next descriptive pull request message.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**